tmt/export.py

```diff
@@ -157,9 +171,9 @@ def enumerate_content(content):
         '<h2>Expected Result</h2>'))) + sorted_test)
     expect = ''.join([f"{v[1]}" for v in sorted_expect])
 
-    def check_section_exists(text):
+    def check_section_exists(text: str) -> str:
         try:
-            return sections_headings[text][0][1]
+            return cast(str, sections_headings[text][0][1])
```

Comment:
Same as above, IIUIC, `sections_heading[text][0][1]` is `Union[int, str]`, therefore full `str()` would be better. Or amend the type, if this particular item is always a string, it might be better to use a more fitting type.

Reply:
solved with conversion to `str`, make sense, thanks!